src/main/java/simpaths/data/Parameters.java

```diff
@@ -1911,11 +1911,17 @@ public static TreeMap<Integer, String> calculateEUROMODpolicySchedule(Country co
         MultiKeyCoefficientMap currentEUROMODpolicySchedule;
 
         if (trainingFlag) {
-            currentEUROMODpolicySchedule = ExcelAssistant.loadCoefficientMap("input" + File.separator + "EUROMODoutput" + File.separator + "training" + File.separator + EUROMODpolicyScheduleFilename + ".xlsx", country.toString(), 1, 3);
-        } else {
-            currentEUROMODpolicySchedule = ExcelAssistant.loadCoefficientMap("input" + File.separator + EUROMODpolicyScheduleFilename + ".xlsx", country.toString(), 1, 3);
+            File trainingSchedule = new File("input" + File.separator + "EUROMODoutput" + File.separator + "training" + File.separator + EUROMODpolicyScheduleFilename + ".xlsx");
+            File runSchedule = new File("input" + File.separator + EUROMODpolicyScheduleFilename + ".xlsx");
+            try {
+                FileUtils.copyFile(trainingSchedule, runSchedule);
```

Comment:
Just noticed that this will always copy the policy schedule if training flag is true. So this scenario is possible:

- User runs GUI with only training data
- User modifies policy schedule in GUI to change years
- User runs model
- Model then loads `calculateEUROMODpolicySchedule`, tests whether training flag is true, and if so overwrites user's policy file

Ideally, the training policy schedule file should *always* be loaded automatically at the very start if only training data is available, and this step should *only* occur before the user specifies the file.

Or, this file never gets used and if the policy schedule file doesn't match it checks and errors out?